### ISSUE TITLE
Error in error message...

### ISSUE THREAD

**Issue by iherman** [w3c/epubcheck#1344]
Consider this in the package file:

```xml
  <manifest>
    <item id="content_001" properties="scripted" fallback="#fallbackid" href="content_001.xhtml" media type="application/xhtml+xml" />
    <item id="fallbackid" href="content_fallback.xhtml" media-type="application/xhtml+xml" />
    <item id="nav" properties="nav" href="nav.xhtml" media-type="application/xhtml+xml" />
  </manifest>
```

Epubcheck flags an error because it should be `fallback="fallbackid"`. However, the error message says:

```
ERROR(RSC-005): [...]: Error while parsing file: value of attribute "fallback" is invalid; must be an XML name without colons
```

The problem is not a colon, the problem is a hash mark...

**Comment by mattgarrish** [comment]
I think this has been raised before. Epubcheck isn't telling you what you did wrong, as it only knows the pattern doesn't match. It's telling you what it expects, a non-colonized XML name (NCName).

**Comment by rdeltour** [comment]
Yeah, this message comes from the RelaxNG schema validator (Jing). We might be able to intercept it and reword it, if anyone can suggest a better wording.

**Comment by rdeltour** [comment]
I think it is a catch-all. I'll have to double check.

**Comment by rdeltour** [comment]
Yeah, this is a catch all, so it's difficult to intercept particular cases when this is raised to reword the message depending on the context.

I'm open to suggestion for something better than "an XML name without colons", but moving this out of v5.0.0 for now.

**Comment by danielweck** [comment]
Copying this here from a recent private discussion:

The French translation is indeed faithful to the canonical English source ... a bit ambiguous/confusing to the end user, but technically correct :)

ENGLISH from upstream project:

https://github.com/relaxng/jing-trang/blob/84ec6ad578d6e0a77342baa5427851f98028bfd8/mod/pattern/src/main/com/thaiopensource/relaxng/pattern/resources/Messages.properties#L81
...
https://github.com/relaxng/jing-trang/blob/84ec6ad578d6e0a77342baa5427851f98028bfd8/mod/xsd-datatype/src/main/com/thaiopensource/datatype/xsd/resources/Messages.properties#L34
+
https://github.com/relaxng/jing-trang/blob/84ec6ad578d6e0a77342baa5427851f98028bfd8/mod/xsd-datatype/src/main/com/thaiopensource/datatype/xsd/resources/Messages.properties#L47

FRENCH from EPUBCheck itself:

https://github.com/w3c/epubcheck/blob/55aad60015ac644a04f7e2648d9969f80d5922b6/src/main/resources/com/thaiopensource/relaxng/pattern/resources/Messages_fr.properties#L87
...
https://github.com/w3c/epubcheck/blob/55aad60015ac644a04f7e2648d9969f80d5922b6/src/main/resources/com/thaiopensource/datatype/xsd/resources/Messages_fr.properties#L40
+
https://github.com/w3c/epubcheck/blob/55aad60015ac644a04f7e2648d9969f80d5922b6/src/main/resources/com/thaiopensource/datatype/xsd/resources/Messages_fr.properties#L53

**Comment by gautierchomel** [comment]
I ran into this misleading message for an ID starting with a number and was unable to identify the error. 

I suggest naming the expected standard so one can search and find it.

"XML Namespace NCName compliant is expected"
